infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -42,7 +42,7 @@
 
     public BaseServer(@Nonnull ServerContext context) {
         serverContext = context;
-        executor = serverContext.getExecutorService(serverContext.getBaseServerThreadCount(), "baseServer-");
```

Comment:
@Maithem are the thread prefixes always hardcoded, or should they use the getPrefix option from CLI args